bamazonCustomer.js

```diff
@@ -0,0 +1,76 @@
+var mysql = require("mysql");
+var inquirer = require("inquirer");
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+
+  user: "root",
+
+  password: "Oak04k88",
```

Comment:
It could make for a better experience for the user to include a process.env variable that gets assigned in the command line so that the user doesn't have to alter the file to make it work for their sql setup.